BackendNew/models/students.js

```diff
@@ -602,9 +602,26 @@ exports.getStudentById = (studentId, sendStudent) => {
                               companiesMap.get(value.batchId).push(value.Name);
                             });
                             for (let batch of batchesMap.keys()) {
-                              batchesMap
-                                .get(batch)
-                                .push({ companies: companiesMap.get(batch) });
+                              var val = batchesMap.get(batch);
+                              var cv;
+                              var stratDate;
+                              var selectedCompany;
+                              val.forEach(value => {
+                                if (value.cv) {
+                                  cv = value.cv;
+                                } else if (value.dateOfStart) {
+                                  stratDate = value.dateOfStart;
+                                } else if (value.selectedCompany) {
+                                  selectedCompany = value.selectedCompany;
+                                }
+                              });
```

Comment:
Can't there be any instance where all cv, dateOfStart, selectedCompany  has value?

Reply:
No. That's why I used this.

Reply:
Got it.